### ISSUE TITLE
Animxplay down

### ISSUE THREAD

**Issue by zezo357** [justfoolingaround/animdl#204]
Just wanted to let you know.

Animxplay website has shutdown, I guess for the package I guess it should not be the default provider and I guess removed from the package 😅

Thank you for the awesome package ❤️

**Comment by justfoolingaround** [comment]
In favor of losing Animixplay, the project has been updated such that AllAnime is no longer a slow provider.

AllAnime now provides ***most likely* highest quality** URL as soon as possible without worrying much about scraping more. However, for those of us who want maximum URLs anyway, you can always use the configuration. 

```yml
superanime:
  type_of: "sub"
  return_all: false
```
`type_of` should be pretty self-explanatory. This is added so that AllAnime no longer scrapes all the types and makes itself slow. This can be either `sub` or `dub`, usually.

`return_all` set to `true` will return everything. This is for developers to test out the capabilities of the provider.

In conclusion, with demise of Animixplay, the project has switched to equally if not more powerful provider which might even be better. The code of `animixplay` will not be removed from the codebase until a proper `animdl-archive` or similar repository is created for it.

**What if AllAnime shuts down?**

If a better provider is available within the project, that will be used. Else, a new provider or a new algorithm which is expected to be **better** than the currently active default provider will be used. The default provider will never fail the user.
